root/user/UserProfile.js

```diff
@@ -560,12 +627,45 @@ const UserProfileStatistics = ({
           </tr>
         </tfoot>
       </table>
+
+      <table
+        className="statistics"
+        title={l('This table shows a summary ' +
+                 'of entities added by this editor.')}
+      >
+        <thead>
+          <tr>
+            <th colSpan="2">
+              {exp.l('Added entities')}
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {Object.keys(ADDED_ENTITIES_TYPES)
+          .map(type => [type, ADDED_ENTITIES_TYPES[type]()])
+          .sort((a, b) => a[1] > b[1] ? 1 : -1)
```

Comment:
The `compare` function might be better here, 'cause it uses the `Intl.Collator` API for language-specific comparisons. You've already imported it but haven't used it, did it not work?
```suggestion
          .sort((a, b) => compare(a[1], b[1]))
```

Reply:
It was probably lost during one rebase, I will rebase & push with it fixed